tom/bot.py

```diff
@@ -84,19 +91,42 @@ def ping_reviewer(self, pr):
 
     def review(self, pr):
         tom = self.username
-        if tom not in pr.maintainers:
+        log.info("Reviewing: {}".format(pr.title))
+
+        if tom in pr.denials:
+            log.info("I've already denied this PR")
             return
-        if tom in pr.approvals:
+
+        log.debug("E-mails: {}".format(pr.emails))
+        bad_emails = set()
+        for email in pr.emails:
+            hash = email_sha256(email)
+            log.debug("{}: {}".format(email, hash))
+            if email in self.banned_emails or hash in self.banned_emails:
+                log.info("Found banned email: " + email)
+                bad_emails.add(email)
+
+        obfuscated = [f"{e[0]}***@{e[e.index('@')+1:]}" for e in bad_emails]
+        bad_emails = ",".join(obfuscated)
```

Comment:
What about `", "` for slightly easier readability? :^)

Reply:
Good suggestion, feel free to open another PR! :)